# Service to test a log file against a singulation detection model
string logFileName         #Name of the log file to test (relative to the log folder)
---
int64 label		   #Classification class
int64 ret
string msg